### ISSUE TITLE
Crash when pasting text

### ISSUE THREAD

**Issue by ilor** [microsoft/terminal#13263]
### Windows Terminal version

1.14.1451.0

### Windows build number

10.0.19044.1706

### Steps to reproduce

Pasted a multi-line text into the terminal (nothing excessive, 5 lines 144 chars each, ascii)

### Actual Behavior

Crashed all tabs with no error message. The multi-line paste warning did not appear (it was a multi-line paste).

I can't reliably reproduce the issue (pasting the exact same text worked OK on a retry). I get that this is not very actionable, but that's all I have at the moment. If there's some crash dump path I can look into, or settings to make future crashes dump some info,  I'd appreciate hints.

**Comment by zadjii-msft** [comment]
Can you try following the steps in [this post](https://www.meziantou.net/tip-automatically-create-a-crash-dump-file-on-error.htm) to set up automatic crash dumps/? (For more info, see [this link](https://docs.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps)) If that works, then you should be able to automatically get a `.dmp` of the terminal when it crashes. Then, can you zip that dump up and send it to us, so we can investigate? Thanks!

**Comment by ilor** [comment]
I definitely will, thanks!

**Comment by ilor** [comment]
Interesting, I must've followed that exact guide some time in the past and long since forgotten about it -- but it seems I had this active, and I do have a WindowsTerminal.exe dump. It seems to correspond, modification-time-wise, to when I had the crash.

The file is 17MB, 2Mb compressed. Should I just attach it here?

**Comment by lhecker** [comment]
@ilor Such dumps can contain private data (like usernames and other bits of information).
Could you send the dump via mail to `<my github username>@microsoft.com`? I'll then forward it to the rest of the team. 🙂
(An informal mail is absolutely fine.)

**Comment by zadjii-msft** [comment]
> tl;dr:
> * It's an unhandled error in `Windows_UI_Xaml!DirectUI::ErrorHelper::ProcessUnhandledError`
> * The `HRESULT m_ErrorCode` passed to that function is 0x80004005 (Unspecified error)
> * The `m_description` string for the error has been stripped from the MiniDump 😿
> * Failure bucket – no hits: `a14d2eeb-64f1-cf9a-221e-7e98eff63046`

Maybe there's a `!xamlstowed` exception in there, but it sounds like the dump unfortunately didn't have anything useful to work with. If you can get a reliable repro at all, then trying to repro this with a Feedback Hub recording might be useful. That sometimes catches extra information. (The bot is about to recommend the Terminal area path in feedback hub, but I might recommend
![image](https://user-images.githubusercontent.com/18356694/173402511-1534089d-07e8-41f4-b25d-75e3e5f1ce4c.png) 
instead)
 
 /feedback

**Comment by zadjii-msft** [comment]
I'm gonna close this as an intermittent failure. If you start seeing this regularly again, let us know and we can reopen and investigate more. Thanks!
